core/src/main/scala/org/apache/spark/scheduler/TaskScheduler.scala

```diff
@@ -66,10 +66,6 @@ private[spark] trait TaskScheduler {
    */
   def killTaskAttempt(taskId: Long, interruptThread: Boolean, reason: String): Boolean
 
-  // Kill all the running task attempts in a stage.
-  // Throw UnsupportedOperationException if the backend doesn't support kill tasks.
-  def killAllTaskAttempts(stageId: Int, interruptThread: Boolean, reason: String): Unit
```

Comment:
Does this is a break change?

Reply:
this is private API

Reply:
Right, the whole `TaskScheduler` is private.

Reply:
Got it.